packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -789,11 +789,14 @@ const Account = Backbone.Model.extend(
         options.newsletters = newsletters;
       }
 
-      return this._fxaClient.sessionVerifyCode(
-        this.get('sessionToken'),
-        code,
-        options
-      );
+      return this._fxaClient
+        .sessionVerifyCode(this.get('sessionToken'), code, options)
+        .then(() => {
+          // If the promise resolves without error, then the code was correct and the
+          // verified flag can be set to true. If verified is not set, the user will
+          // likely be directed to the signin.
+          this.set('verified', true);
+        });
```

Comment:
Previously this was returning a Promise (I guess whatever `.sessionVerifyCode()` returned), but now I think now that there is a chained `.then()`, we're returning a Promise'd `undefined`?

Not sure if it matters or if anybody was using the returned value from `this._fxaClient.sessionVerifyCode()` previously; or should we return the response from `.sessionVerifyCode()`? 🤷